src/Pages/shop/Shop.js

```diff
@@ -13,10 +13,10 @@ class Shop extends Component {
   }
 
   componentDidMount = () => {
-    fetch("http://localhost:3000/data/product/product.json")
+    fetch("http://10.58.7.172:8000/products?category=1")
```

Comment:
```suggestion
    fetch(`{API}/products?category=1`)
```
src 폴더 내에 config.js 파일 만든 뒤 api 주소 일괄 관리해주세요!

Reply:
반영했습니다